src/dataset-builder/ConceptSetCreator.ts

```diff
@@ -11,17 +11,26 @@ export type ConceptSetCreatorProps = {
   readonly onStateChange: OnStateChangeHandler;
   readonly dataset: DatasetModel;
   readonly conceptSetUpdater: Updater<ConceptSet[]>;
+  readonly cart: Concept[];
 };
 
+// featureValueGroupName represents a domain name
+// this works because the only concepts passed in are also domains
+// such as Condition
 export const toConceptSet = (concept: Concept): ConceptSet => {
   return {
     name: concept.name,
+    concept,
```

Comment:
concept includes concept.name so this is storing duplicated data, I will change it to not include the name field and just call concept.name

Reply:
Or I could make Concept not extend anything